app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    main_file, for_copy_file = parts[1], parts[2]
```

Comment:
Unpack parts without use of indexes, it will be safe to to so since you checked that there are three elements in the `parts` list
```suggestion
    command_name, main_file, for_copy_file = parts
```